### ISSUE TITLE
Не срабатывает адаптивность страничек

### ISSUE THREAD

**Issue by vpougatchev60** [olendeer/bha#2]
Если размер окна становится меньше любого из размеров 1024 х 600, то верстка должна автоматически меняться на мобильный вариант. Этого не происходит - см. скриншоты.

![image](https://user-images.githubusercontent.com/79328371/108920275-3ca1d280-75e9-11eb-8b16-6523e6f7c3a8.png)

![image](https://user-images.githubusercontent.com/79328371/108920359-63f89f80-75e9-11eb-89d3-56f9b2d6e607.png)

**Comment by olendeer** [comment]
Пока что в разработке только десктоп версия, моб версию ещё не обсуждали, как я понимаю, начнем её после завершения дестоп версии

**Comment by olendeer** [comment]
Моб версия запланирована на 5й этап разработки

**Comment by klymeniuk** [comment]
Да, согласовано выполнение адаптивности после основных работ по десктопной версии.
